File: c-007/main.r
isPrime <- function(number) {
    limit <- number
    i <- 2
    while (i < limit) {
        if (number %% i == 0) {
            return(FALSE)
        } else {
            limit <- floor(number / i)
        }
        i <- i + 1
    }
    TRUE
}

primeAt <- function(at) {
    loopen <- 0
    number <- 1
    while (loopen < at) {
        number <- number + 1
        if (isPrime(number)) {
            loopen <- loopen + 1
        }
    }
    number
}

print(primeAt(10001))

File: c-007/main.go
package main

import (
	"fmt"
	"math"
)

func isPrime(number int) bool {
	limit := number
	for i := 2; i < limit; i++ {
		if number%i == 0 {
			return false
		} else {
			limit = int(math.Floor(float64(number / i)))
		}
	}
	return true
}

func primeAt(at int) int {
	loopen, number := 0, 1
	for loopen < at {
		number++
		if isPrime(number) {
			loopen++
		}
	}
	return number
}

func main() {
	fmt.Println(primeAt(10001))
}
